components/panel/body-toggle.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * WordPress dependencies
+ */
+import Dashicon from 'components/icon-button';
+
+function PanelBodyToggle( { opened, label, onClick } ) {
+	return (
+		<button className="components-panel__body-toggle" onClick={ onClick }>
```

Comment:
Out of curiosity, how difficult would it be to render and style this as an `IconButton` and avoid the need for a specialized `PanelBodyToggle` altogether?